pi/heartbeat.py

```diff
@@ -102,6 +103,33 @@ def ping(host):
     logging.info('returned: {}'.format(returncode))
     return returncode == 0
 
+def parse_link_quality_from_iwconfig_output(iwconfig_output_text):
+    lines = iwconfig_output_text.split('\n')
+    for i in range(0, len(lines)):
+        # note: hardcoded wireless network interface name
+        if ('wlan0' in lines[i]) and (i+5 < len(lines)):
```

Comment:
Instead of hardcoding it here, you could do it in the command itself and only have to deal with a single line (https://manpages.debian.org/buster/wireless-tools/iwconfig.8.en.html)

On my computer:
![image](https://user-images.githubusercontent.com/1490437/103598702-249ab600-4eb8-11eb-9b5d-dc06e105f627.png)

Reply:
Ah yeah good idea. I'll do this.

Reply:
done.